modules/core/src/main/java/org/apache/ignite/cluster/BaselineNode.java

```diff
@@ -67,4 +70,23 @@
      * @return All node attributes.
      */
     public Map<String, Object> attributes();
+
+    /**
+     * Gets affinity attribute by name.
+     *
+     * @param name of the attribute.
+     * @return Affinity attribute.
+     */
+    public default String affinityAttribute(String name) {
+        return (String)attributes().get(AFFINITY_ATTR_PREFIX + '.' + name);
```

Comment:
If we wand to avoid creating new entity we'll have to tell user to use specific prefix for affinity attributes. It might be easier for us but not for users